README.md

```diff
@@ -36,33 +36,33 @@ yarn add @femessage/update-popup
 
 ## Usage
 
-你需要通过环境变量 `UPDATE_POPUP_VERSION` 来传入版本号，后续每次迭代更新只需要修改比当前大的版本号即可。
+You need to use environment variables `UPDATE_POPUP_VERSION` To pass in the version number, when iteratively updating, modify version number greater than the previous version number
```

Comment:
`You need to set environment variables `UPDATE_POPUP_VERSION` To pass in the version number, when iteratively updating, modify version number greater than the previous version number`